### PR TITLE
doc: add minutes for meeting 17 dec 2018

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2018-12-17.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+* Joel Chen (@jchip)
+* Wes Todd (@wesleytodd)
+* Michael Dawson( @mhdawson)
+* Ankur Oberoi (@aoberoi)
+* Antonio Gonzalez (@bluesockets)
+* Viktor Vershanskij (@wentout)
+* Pavel Vostrikov (@vostrik)
+* Dominykas Blyžė (@dominykas)
+* Potham (@potham) 
+
```
Comment (ljharb) [inline meetings/2018-12-17.md]:
```suggestion
 * Jordan Harband (@ljharb)
 
```

File: meetings/2018-12-17.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  * Onboarding to enable people to get engaged with the work under this effort.
+
+  * Process to get sub-teams working
+
+  * Extending Canary in the Gold mine.
+
+* Agreed next steps
+
+  * Defining so key packages and friendlies - Express ecosystem 
+    * Wes to open issue to get this started
```
Comment (wesleytodd) [inline meetings/2018-12-17.md]:
```suggestion
    * Wes to open issue to get this started [#105](https://github.com/nodejs/package-maintenance/issues/105)
```

File: meetings/2018-12-17.md (hunk 1 above)
Comment (Eomm) [inline meetings/2018-12-17.md]:
```suggestion
* Manuel Spigolon (@eomm)
```
I was on youtube channel 😃 (the one who ad an answer at the end if you remind).
I will join to zoom next time 👍

**Comment by mhdawson** [comment]
@ljharb  @Eomm comments addresssed

**Comment by mhdawson** [comment]
@mcollina can you review/approve

**Pull request merged**